sematic/db/models/run.py

```diff
@@ -139,6 +139,7 @@ class Run(Base, JSONEncodableMixin, HasExternalJobsMixin):
         types.JSON(), nullable=True, info={JSON_KEY: True}
     )
     cache_key: Optional[str] = Column(types.String(), nullable=True)
+    user_id: Optional[str] = Column(types.String(), nullable=True)
```

Comment:
Please see [this comment](https://github.com/sematic-ai/sematic/pull/602#discussion_r1117467549).

Reply:
Done.